hack/testdata/multi-resource.json

```diff
@@ -0,0 +1,60 @@
+{
+   "kind":"List",
+   "apiVersion":"v1",
+   "items":[
+      {
+         "kind":"ReplicationController",
+         "apiVersion":"v1",
+         "metadata":{
+            "name":"mock",
+            "labels":{
+               "name":"mock"
```

Comment:
I know this is only a test, but please don't copy the practice of using "name" as a label key:
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md

I suggest "app". (Here, and in all other labels and selectors.)